plugins/lookup/onepassword.py

```diff
@@ -472,55 +483,77 @@ def _parse_field(self, data_json, field_name, section_title=None):
         return ""
 
     def assert_logged_in(self):
-        args = ["account", "list"]
-        if self.subdomain:
-            account = "{subdomain}.{domain}".format(subdomain=self.subdomain, domain=self.domain)
-            args.extend(["--account", account])
+        if self.service_account_token:
+            # maybe not necessary
+            args = ["account", "get"]
+            environment_update = {"OP_SERVICE_ACCOUNT_TOKEN": self.service_account_token}
+            rc, out, err = self._run(args, ignore_errors=False, environment_update=environment_update)
 
-        rc, out, err = self._run(args)
+            return not bool(rc)
 
-        if out:
-            # Running 'op account get' if there are no accounts configured on the system drops into
-            # an interactive prompt. Only run 'op account get' after first listing accounts to see
-            # if there are any previously configured accounts.
-            args = ["account", "get"]
+        else:
+
```

Comment:
This is still probably necessary as it checks that the token is valid.

The `else` can be removed.